templates/components/colors.html

```diff
@@ -0,0 +1,81 @@
+{% block colors-content %}
+
+<h3>Color usage</h3>
+
+{% block colors-intro %}
+    <p>When using a colour in a sass file, please try to use a descriptive variable name not the colour name itself. This will allow for easy reskinning in future.</p>
+{% endblock colors-intro %}
+
+{% block main-colors %}
+    <h3>Main colors</h3>
+
+    {% for color in colors %}
+        <div class="color {{ loop.key }}">
+            <p class="color-name">{{ loop.key }}</p>
+            <div class="color-block"></div>
+            <p class="hexcode">{{ color }}</p>
+        </div><!-- color -->
+    {% endfor %}
+{% endblock main-colors %}
+
+{% block greys %}
+    <h3>Greys</h3>
+
+    <div class="color grey-1">
+        <p class="color-name">grey-1</p>
+        <div class="color-block"></div>
+        <p class="hexcode">Lighten 45%</p>
+    </div><!-- color -->
```

Comment:
a line break between each block would be nice, I think then the compiled one would follow it too